internal/db/repo.go

```diff
@@ -429,24 +429,26 @@ func (repo *Repository) UpdateSize() error {
 	return nil
 }
 
-// ComposeMetas composes a map of metas for rendering external issue tracker URL.
+// ComposeMetas composes a map of metas for rendering SHA1 URL and external issue tracker URL.
 func (repo *Repository) ComposeMetas() map[string]string {
-	if !repo.EnableExternalTracker {
-		return nil
-	} else if repo.ExternalMetas == nil {
-		repo.ExternalMetas = map[string]string{
-			"format": repo.ExternalTrackerFormat,
-			"user":   repo.MustOwner().Name,
-			"repo":   repo.Name,
-		}
+	if repo.ExternalMetas == nil {
+		repo.ExternalMetas = make(map[string]string)
+	}
+
+	repo.ExternalMetas["user"] = repo.MustOwner().Name
+	repo.ExternalMetas["repo"] = repo.Name
```

Comment:
Use the result of [`repo.Link`](https://github.com/gogs/gogs/blob/211a1394b86dafc90d73d5d622ca85686c2fb5f6/internal/db/repo.go#L544-L546) is better here, that's all we want.
```suggestion
	repo.ExternalMetas["repoLink"] = repo.Link()
```